lib/membrane_ffmpeg_swresample_plugin/converter.ex

```diff
@@ -168,10 +168,13 @@ defmodule Membrane.FFmpeg.SWResample.Converter do
         {[end_of_stream: :output], %{state | queue: <<>>}}
 
       converted ->
-        converted_frames_count =
-          byte_size(converted) / RawAudio.frame_size(state.output_stream_format)
+        input_frame_size = RawAudio.frame_size(state.input_stream_format)
 
-        {state, out_pts} = update_pts_queue(state, converted_frames_count)
+        converted_frames =
+          byte_size(converted) / RawAudio.frame_size(state.output_stream_format) *
+            (input_frame_size * state.input_stream_format.sample_rate)
+
+        {state, out_pts} = update_pts_queue(state, converted_frames)
```

Comment:
- this is the same thing we do in `handle_process`, maybe wrap in a function?
- `converted_frames` is not the right name anymore, maybe something like `output_duration`?